app/pulp/app/serializers/progress.py

```diff
@@ -0,0 +1,57 @@
+from rest_framework import serializers
+
+from pulp.app import models
+from pulp.app.serializers import ModelSerializer
+
+
+class ProgressReportSerializer(ModelSerializer):
+
+    _href = serializers.HyperlinkedIdentityField(
```

Comment:
`_href` should be provided by the ModelSerializer, and will just work magically. The reason the other serializers have to define this is that their `lookup_field` is something other than `id`.